app/routes/img.$slug/utils.server.tsx

```diff
@@ -19,11 +18,15 @@ export async function createOgImageSVG(request: Request) {
   let searchParams = new URLSearchParams(requestUrl.search);
   let siteUrl = requestUrl.protocol + "//" + requestUrl.host;
 
-  let { title, displayDate, authors } = getDataFromParams(
+  let { title, displayDate, authors, ogImage } = getDataFromParams(
     siteUrl,
     searchParams,
   );
 
+  const backgroundImage = ogImage
+    ? `url(${siteUrl}${ogImage})`
+    : `url("data:image/png;base64,${arrayBufferToBase64((await import("./social-background.png?arraybuffer")).default)}")`;
```

Comment:
Total nit - take it or leave it lol

```suggestion
  let backgroundImage: string;
  if (ogImage) {
    backgroundImage = `url(${siteUrl}${ogImage})`;
  } else {
    let socialBackground = await import("./social-background.png?arraybuffer");
    let base64 = arrayBufferToBase64(socialBackground.default);
    backgroundImage = `url("data:image/png;base64,${base64}")`;
  }
```

Reply:
Works for me, I was worried this was a bit too 1-liner-y